site/app/libraries/database/PostgresqlDatabaseQueries.php

```diff
@@ -25,7 +25,16 @@
 class PostgresqlDatabaseQueries extends DatabaseQueries{
 
     public function getUserById($user_id) {
-        $this->course_db->query("
+        $details = array();
+
+        $this->submitty_db->query("SELECT * FROM users WHERE user_id=?", array($user_id));
+
+        if (count($this->submitty_db->rows()) > 0) {
+            $details = array_merge($details, $this->submitty_db->row());
+        }
```

Comment:
Let's change this to be:
```
if (this->submitty_db->getRowCount() === 0) {
    return null;
}
$details = $this->submitty_db->row();
```

No need to waste time checking the course DB if they don't exist in the master DB